apps/gitness/src/pages/pull-request/stores/pull-request-store.tsx

```diff
@@ -0,0 +1,255 @@
+import { create } from 'zustand'
+import { produce } from 'immer'
+import { mergePullReqOp, commentStatusPullReq as apiCommentStatusPullReq } from '@harnessio/code-service-client'
+import { CodeCommentState, PullRequestState } from '../types/types'
+import { PullRequestDataState } from '@harnessio/views'
+export const codeOwnersNotFoundMessage = 'CODEOWNERS file not found'
+export const codeOwnersNotFoundMessage2 = `path "CODEOWNERS" not found`
+export const codeOwnersNotFoundMessage3 = `failed to find node 'CODEOWNERS' in 'main': failed to get tree node: failed to ls file: path "CODEOWNERS" not found`
+export const oldCommitRefetchRequired = 'A newer commit is available. Only the latest commit can be merged.'
+export const prMergedRefetchRequired = 'Pull request already merged'
```

Comment:
1. why are these messages hardcoded with so many variants?
2. why are these in the data store? these should be in the `view` layer maybe?

Reply:
yeah so these are the error messages used in the drymerge function which checks the current pr
               ``` err.message === codeOwnersNotFoundMessage ||
                err.message === codeOwnersNotFoundMessage2 ||
                err.message === codeOwnersNotFoundMessage3 ||```